README.md

```diff
@@ -96,41 +74,41 @@ The added property `AndroidManifest.xml`. Note: You should only include the `clo
 
 
 
-### Embedding and transforming images
+### Transform and Optimize Assets
 
 Any image uploaded to Cloudinary can be transformed and embedded using powerful view helper methods:
 
 The following example generates the url for accessing an uploaded `sample` image while transforming it to fill a 100x150 rectangle:
 
     MediaManager.get().url().transformation(new Transformation().width(100).height(150).crop("fill")).generate("sample.jpg")
 
-Another example, embedding a smaller version of an uploaded image while generating a 90x90 face detection based thumbnail: 
+Another example, embedding a smaller version of an uploaded image while generating a 90x90 face detection based thumbnail:
 
     MediaManager.get().url().transformation(new Transformation().width(90).height(90).crop("thumb").gravity("face")).generate("woman.jpg")
 
-You can provide either a Facebook name or a numeric ID of a Facebook profile or a fan page.  
-             
+You can provide either a Facebook name or a numeric ID of a Facebook profile or a fan page.
+
 Embedding a Facebook profile to match your graphic design is very simple:
 
     MediaManager.get().url().type("facebook").transformation(new Transformation().width(130).height(130).crop("fill").gravity("north_west")).generate("billclinton.jpg")
-                           
+
 Same goes for Twitter:
 
     MediaManager.get().url().type("twitter_name").generate("billclinton.jpg")
 
 ### Uploading
```

Comment:
```suggestion
### Uploading Assets
```

Reply:
Changed the two headings to be consistent styles. Using gerunds (which are nouns) rather than verb headings, because all the H2s in the template are also nouns.

@patrick-tolosa - This should be followed at a template level.